### ISSUE TITLE
cloudbuild: ortelius-ms-rpt-app2env needs cloudbuild yaml

### ISSUE THREAD

**Issue by sbtaylor15** [ortelius/ortelius#158]
Need cloudbuild for [ortelius-ms-rpt-app2env](https://github.com/ortelius/ortelius-ms-rpt-app2env).  See [cloudbuild.yaml](https://github.com/ortelius/ortelius-ms-rpt-app2comp/blob/main/cloudbuild/cloudbuild.yaml) for an example.

**Comment by chicks-net** [comment]
@sbtaylor15 the build results haven't shown up in the Discord channel for this after 15 minutes.  Previous builds for this repo seem to only take a minute to complete.  Any ideas?

**Comment by chicks-net** [comment]
I've compared `cloudbuild.yaml` from this repo with the one from `ortelius-ms-rpt-comp2app` which works and found that:

* both have the same errors from `yamllint`: line length, trailing spaces, indentation, brackets
* the only differences are the names of the repos (shown below)

```
$ diff cloudbuild.yaml ../../ortelius-ms-rpt-app2env/cloudbuild/cloudbuild.yaml 
19c19
<       - 'COMPONENT_GITURL=git@github.com:ortelius/ortelius-ms-rpt-comp2app.git'
---
>       - 'COMPONENT_GITURL=git@github.com:ortelius/ortelius-ms-rpt-app2env.git'
21c21
<       - 'COMPONENT_DOCKERREPO=quay.io/ortelius/ms-rpt-comp2app'
---
>       - 'COMPONENT_DOCKERREPO=quay.io/ortelius/ms-rpt-app2env'
```

* I was able to check out the git repo with that URL so that's ok.
* So it doesn't seem like this has anything to do with the file in the repo.
* This must be an issue with the webhooks or cloudbuild config that I don't think I have access to. https://cloud.google.com/build/docs/automating-builds/run-builds-on-github
